packages/waas/src/session/secp256r1.ts

```diff
@@ -1,13 +1,20 @@
 import { ethers } from 'ethers'
 import { Session } from './index'
 import { KeyTypes } from './keyTypes'
+import { SubtleCryptoBackend } from '../subtle-crypto'
 
 import { openDB } from 'idb'
+import { SecureStoreBackend } from '../secure-store'
 
 const idbName = 'seq-waas-session-p256r1'
 const idbStoreName = 'seq-waas-session'
 
-export async function newSECP256R1SessionFromSessionId(sessionId: string): Promise<Session> {
+// TODO: We need to update this to use the secure store backend
```

Comment:
if the CryptoKeyPair type becomes a problem, we could make the type `any` at least on our interface, but we'd need to typecast and confirm inside of the backend implementations.. in case we wanted to handle other kinds of types on the interface methods

Reply:
I've moved to `any` type on the interface and converted R1 implementation to use SecureStoreBackend. KeychainStore won't work with R1 for now, but we can resolve that when R1 is ready for RN.